test/module/test_rules_collections.py

```diff
@@ -66,3 +73,108 @@ def test_fail_sub_properties_run(self):
         matches = []
         matches.extend(self.rules.run(filename, cfn))
         assert len(matches) == 3, 'Expected {} failures, got {}'.format(2, len(matches))
+
+    def test_success_filtering_of_rules_default(self):
+        """Test extend function"""
+        class rule_e0000(CloudFormationLintRule):
+            """Error Rule"""
+            id = 'E0000'
+
+        class rule_w0000(CloudFormationLintRule):
+            """Warning Rule"""
+            id = 'W0000'
+
+        class rule_i0000(CloudFormationLintRule):
+            """Error Rule"""
```

Comment:
I don't think you need this class for the default check?

Reply:
I do.  I need to setup rules with IDs for testing purposes.